.github/workflows/release_kolibri.yml

```diff
@@ -160,6 +179,40 @@ jobs:
       with:
         path: 'dist/${{ matrix.filename }}'
         destination: '${{ secrets.KOLIBRI_PUBLIC_RELEASE_GCS_BUCKET }}/downloads/kolibri/${{ github.event.release.name }}'
+  bck_prerelease_gcs_upload:
+    name: Upload WHL file to Google Cloud Storage for BCK Pre-release
+    runs-on: ubuntu-latest
+    needs: [whl]
+    steps:
+    - name: Download WHL artifact
+      uses: actions/download-artifact@v4
+      with:
+        name: ${{ needs.whl.outputs.whl-file-name }}
+        path: dist
+    - name: Zip whl file
+      run: zip -j dist/kolibri.zip dist/${{ needs.whl.outputs.whl-file-name }}
+    - uses: 'google-github-actions/auth@v2'
+      with:
+        credentials_json: '${{ secrets.GH_UPLOADER_GCP_SA_CREDENTIALS }}'
+    - name: 'Set up Cloud SDK'
+      uses: 'google-github-actions/setup-gcloud@v2'
+    - name: Upload to BCK bucket
+      uses: 'google-github-actions/upload-cloud-storage@v2'
+      with:
+        path: 'dist/kolibri.zip'
+        destination: '${{ secrets.BCK_PRERELEASE_BUILD_ARTIFACT_GCS_BUCKET }}'
+        parent: false
+    - name: Unzip content static files from whl file
+      run: |
+        unzip -j dist/${{ needs.whl.outputs.whl-file-name }} 'kolibri/core/content/static/*' -d static
+        rm static/*.file_size
+        # Ungzip all .gz files in the static folder
+        for f in static/*.gz; do gunzip -f $f; done
+    - name: Upload content static files to BCK bucket
+      uses: 'google-github-actions/upload-cloud-storage@v2'
+      with:
+        path: 'static'
+        destination: '${{ secrets.STUDIO_BCK_CONTENT_STATIC_BUCKET }}'
```

Comment:
Yes - the docs are highly confusing, but this is also why I unzipped all the static files into the root of the current directory, so that I could just say "upload this folder". I am fully ready for my having misunderstood the docs again, again, again.